tests/dummy/app/templates/demos/sl-date-picker.hbs

```diff
@@ -52,6 +48,10 @@
         Days of the week that should be disabled. Values are 0 (Sunday) to 6 (Saturday). Multiple values should be comma-separated.
     {{/property-text}}
 
+    {{#property-text name="disabled" type="string" default="false"}}
+        When true, the datepicker will not be displayed.
```

Comment:
Same comment about this property in the component - make sure to synchronize whatever the new property description becomes.